template-for-repository/proofs/Makefile.common

```diff
@@ -233,6 +235,9 @@ HARNESS_GOTO ?= $(GOTODIR)/$(HARNESS_FILE)
 PROJECT_GOTO ?= $(GOTODIR)/$(PROJECT)
 PROOF_GOTO ?= $(GOTODIR)/$(PROOF)
 
+ARPA_BLDDIR ?= $(abspath $(PROOFDIR)/arpa_cmake)
```

Comment:
* I notice that all other directory names in this file have been hardcoded: (`PROOF_SOURCE = $(CBMC_ROOT)/sources`,  `GOTODIR ?= $(PROOFDIR)/gotos`, `...`). As a result, I have not made `arpa_cmake` a variable that can be overridden to stay consistent with the rest of the file. Furthermore, this makes it easier to gitignore the arpa build directory, I believe.
Nevertheless, I can make the adjustment if you prefer.

* I have added `$(abspath ...)`.